### PR TITLE
Fix typo

### PR DESCRIPTION
**Pull request by dnicolson**
What does this implement/fix? Explain your changes.
---------------------------------------------------
This pull request fixes a small typo.

### CODE REVIEW

**Comment by Jason-Morcos** [comment]
Good catch! Please point this PR at dev branch. Master is reserved for approved App Store releases

File: Source/SPPreferencesUpgrade.m [hunk 1]
```diff
@@ -79,7 +79,7 @@ void SPApplyRevisionChanges(void)
 	
 	// This is how you add release notes and run specific migration steps
 	if (recordedVersionNumber < 2061) {
-		[importantUpdateNotes addObject:NSLocalizedString(@"There is a new option in Preferences->Alerts & Logs: \"Show warning before executing a query\". When enabled, you will be prompted to confirm that you want to execte an SQL query or edit a row.", @"Short important release note for new option in Preferences->Alerts & Logs")];
+		[importantUpdateNotes addObject:NSLocalizedString(@"There is a new option in Preferences->Alerts & Logs: \"Show warning before executing a query\". When enabled, you will be prompted to confirm that you want to execute an SQL query or edit a row.", @"Short important release note for new option in Preferences->Alerts & Logs")];
```
Comment (Jason-Morcos) [inline Source/SPPreferencesUpgrade.m:82]: Shouldn't it be "execute _a_ SQL query"? If we're gonna fix the spelling error we can fix the grammar one too!

Comment (samuelbradshaw) [reply Source/SPPreferencesUpgrade.m:82]: It depends on if you pronounce it "Sequel" or "S-Q-L".

Comment (dnicolson) [reply Source/SPPreferencesUpgrade.m:82]: It's "My Ess Que Ell" according to https://dev.mysql.com/doc/refman/8.0/en/what-is-mysql.html.

Comment (Jason-Morcos) [reply Source/SPPreferencesUpgrade.m:82]: Fascinating, I've always pronounced it "Sequel"! Thanks for the link!

**Review by Jason-Morcos — APPROVED** [review]

**Pull request merged**